.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: go
 go:
   - 1.4
```

Comment:
I would just drop this old version of Go. Maybe just replace it with 1.9 which I think is still the default version used on the old SLE12

Reply:
Alright, I dropped 1.4 for 1.9.x